### PR TITLE
[FRONTEND][TENSORFLOW] Optimized tensorflow testcases

### PR DESCRIPTION
**Pull request by joyalbin**
This PR is to remove the code duplication in tensoflow testcases

### CODE REVIEW

**Comment by srkreddy1238** [comment]
Suggest to wait a while until #1505 is merged. 
Above PR offers unified function which can be used along with local optimizations this PR offers.

**Comment by tqchen** [comment]
please resync the changes against current master as in https://github.com/dmlc/tvm/issues/1394

**Comment by srkreddy1238** [comment]
@joyalbin thanks for improving the test cases.

Please note using #1505 we could improve further with common check_function API.
This can be handled outside this PR once the other is merged.

**Comment by joyalbin** [comment]
@srkreddy1238 @yzhliu I have handled the review comments and rebased the PR, 
please help for further review

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 1]
```diff
@@ -1007,11 +822,11 @@ def test_forward_l2_normalize():
         _test_forward_concat_v2()
     test_forward_multi_input()
     test_forward_inception_v3()
-    test_forward_inception_v1()
+    #test_forward_inception_v1()
```
Comment (srkreddy1238) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: Pls remove local changes.

**Comment by yzhliu** [comment]
@srkreddy1238 would you mind take another look and approve if it is good to you?
@joyalbin Please merge from master again.

**Comment by yzhliu** [comment]
Thanks, this is now merged.

**Pull request merged**